source/renderer/app/components/profile/terms-of-use/TermsOfUseForm.js

```diff
@@ -69,7 +68,7 @@ export default class TermsOfUseForm extends Component<Props, State> {
       isSubmitting ? styles.submitButtonSpinning : styles.submitButton,
     ]);
 
-    const checkboxLabel = environment.isEtcApi() ? 'checkboxLabelWithDisclaimer' : 'checkboxLabel';
+    const checkboxLabel = 'checkboxLabel';
```

Comment:
@daniloprates we don't need this variable any longer - we can now use directly `messages. checkboxLabel`